docs/api/safety.md

```diff
@@ -62,3 +62,19 @@ By fully disabling safety features, i.e. setting `safety_override = full` the ro
 
 !!! attention 
     The robot may not be able to stop in time if driving at full speed if a cliff is detected!
+
+## Acceleration Limits
+
+The robot exposes its commanding acceleration limits through the `wheel_accel_limit` parameter on the `motion_control` node.
+The velocity commands sent to the wheels will be ramped with the acceleration profile associated with this value.
+The value defaults to its maximum settable `900 mm/s^2`.  If using heavier payloads, it is advisable to decrease acceleration limits.
+
+## EStop
+
+The robot exposes a service with name `e_stop` to turn off the robot's wheels.  The robot will not respond to velocity
+commands when this service is commanded on.  See [EStop.srv](https://github.com/iRobotEducation/irobot_create_msgs/blob/main/srv/EStop.srv)
```

Comment:
The sentence is a little bit confusing: "a service to turn off wheels"... " will not respond to velocity commands when is turned on".
I would be more explicit on what passing true or false in the request means